guides/source/association_basics.md

```diff
@@ -1758,6 +1762,7 @@ Controls what happens to the associated objects when their owner is destroyed:
 
 * `:destroy` causes all the associated objects to also be destroyed
 * `:delete_all` causes all the associated objects to be deleted directly from the database (so callbacks will not execute)
+* `:destory_async`: when the object is destroyed, an `ActiveRecord::DestroyAssociationAsyncJob` job is enqueued to which will call destroy on its associated objects. ActiveJob must be setup for this to work.
```

Comment:
In our docs, when we talk about a library we tend to use the library name rather than the module name (ie Active Job over ActiveJob).

```suggestion
* `:destory_async`: when the object is destroyed, an `ActiveRecord::DestroyAssociationAsyncJob` job is enqueued to which will call destroy on its associated objects. Active Job must be setup for this to work.
```